int32 frame_rate
---
bool result